tests/integration/plugins/test_signer.py

```diff
@@ -263,6 +264,7 @@ async def test_signer_plugin_reviewer_verifications(
 
     await upload_records(editor_client, 5)
 
+    # TODO: this doesn't raise. Why not?
```

Comment:
I'm guessing this has something to do with some permission configured for the dev instance but not for the local instance.

Reply:
Exactly

Reply:
Any ideas on what we should do about this? Should we skip some of these assertions if we're running on DEV?

Reply:
What about:
- inspect the `to_review_enabled` for the collection in the root URL capabilities
- skip certain assertions (if that doesn't bloat the test too much) if the flag is False
?